modules/programs/kitty.nix

```diff
@@ -89,7 +116,14 @@ in {
 
       ${toKittyKeybindings cfg.keybindings}
 
+      ${toKittyEnv cfg.environment}
+
       ${cfg.extraConfig}
     '';
+
```

Comment:
There is [no documented escaping](https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.env) here, although it does say that kitty will recursively expand `${VAR}`.

My personal config injects the system environment, and since nix-darwin seems to store`$(var)` for recursive expansion and manually expand it later, I [wrote code to translate that format](https://github.com/eraserhd/dotfiles/blob/develop/kitty/default.nix#L5-L22).

I was on the fence as to whether this module module should also do this.  It would definitely be good for me if it did, but I'm not sure whether my use case is common.  Probably the expansion code already exists somewhere.

Reply:
done